docs/crocks/Either.md

```diff
@@ -10,80 +10,84 @@
 --
 ```
 
-`Either` exposes the following constructor and instances:
+`Either` exposes the following functions on the constructor and instance:
 
 | Constructor | Instance |
 |:---|:---|
-| [`Left`](#left), [`Right`](#right), [`of`](#of) | [`inspect`](#inspect), [`value`](#value), [`type`](#type), [`equals`](#equals), [`map`](#map), [`ap`](#ap), [`of`](#of), [`chain`](#chain), [`sequence`](#sequence), [`traverse`](#traverse) |
+| [`Left`](#left), [`of`](#of), [`Right`](#right), [`type`](#type) | [`alt`](#alt), [`ap`](#ap), [`bimap`](#bimap), [`chain`](#chain), [`coalesce`](#coalesce), [`either`](#either), [`equals`](#equals), [`inspect`](#inspect), [`map`](#map),  [`of`](#of), [`sequence`](#sequence), [`swap`](#swap), [`traverse`](#traverse), [`type`](#type) |
```

Comment:
I would say, lets go full on Alphabetical. Capital letters in ASCII come first, so it would be something like:
* `Maybe`: `Just`, `Nothing`, `of`, `type`, `zero`
* `Either`: `Left`, `Right`, `of` `type`

And I think that should do it.